modules/data-store/src/data-store.js

```diff
@@ -0,0 +1,147 @@
+
+import serializer from "abitbol-serializable/lib/serializer";
+import minimatch from "minimatch";
+
+
+/**
+ * Stores project's entities and blobs.
+ */
+class DataStore {
+
+    constructor() {
+        this.entitiesByPath = {};
+        this.entitiesByUuid = {};
```

Comment:
I think you should use `Symbol`s here to avoid access to those properties as they should not be modified from outside of this class

Reply:
It was for the tests, should I put a getter ?

Reply:
You can export symbols and use them in tests :)